README.md

```diff
@@ -57,6 +57,25 @@ NB:
   user (i.e. the application is expected to be served via a reverse proxy)
 
 
+### Docker
+
+* build:
+
+        $ docker build --rm -t naveed .
+
+* run:
+
+        $ docker run -it --rm --name naveed -p8080:8465
+                -v "$(pwd)/config:/app/config" \
```

Comment:
this assumes there's a `config` directory from which we can retrieve our configuration files (customized `naveed.ini` and `tokens.cfg`) - should this perhaps reside within `Dockerfile` instead?